docs/pages/installation.mdx

```diff
@@ -137,13 +137,15 @@ chart.
 
 The Teleport package in Homebrew is not maintained by Teleport and we can't
 guarantee its reliability or security. We recommend the use of our [official
-Teleport packages](https://goteleport.com/download).
+Teleport packages](https://goteleport.com/download). Binaries provided by Homebrew
+are not signed by Teleport that will prevent certain functionality such as 
+using Touch ID or Device Trust with `tsh`.
```

Comment:
```suggestion
Teleport packages](https://goteleport.com/download). Binaries provided by Homebrew
are not signed by Teleport, so features that require signed and notarized binaries
(TouchID, Device Trust) are not available in Homebrew builds.
```

Reply:
Also, not a huge issue, but note that you might confuse people by mentioning device trust in the v11 and v10 backports, since it's only available in 12+

Reply:
Thanks, will take that into account with any backports.